taskcoach/tools/pygettext.py

```diff
@@ -289,13 +304,14 @@ def _get_modpkg_path(dotted_name, pathlist=None):
     extension module.
     """
     # split off top-most name
-    parts = dotted_name.split('.', 1)
+    parts = dotted_name.split(".", 1)
 
     if len(parts) > 1:
         # we have a dotted path, import top-level package
         try:
             file, pathname, description = imp.find_module(parts[0], pathlist)
```

Comment:
why imp.find_module() is not changed.
https://docs.python.org/3.11/library/imp.html :
Deprecated since version 3.3: Use [importlib.util.find_spec()](https://docs.python.org/3.11/library/importlib.html#importlib.util.find_spec) instead unless Python 3.3 compatibility is required, in which case use [importlib.find_loader()](https://docs.python.org/3.11/library/importlib.html#importlib.find_loader). For example usage of the former case, see the [Examples](https://docs.python.org/3.11/library/importlib.html#importlib-examples) section of the [importlib](https://docs.python.org/3.11/library/importlib.html#module-importlib) documentation.